.npmrc

```diff
@@ -0,0 +1 @@
+legacy-peer-deps=true
```

Comment:
We need to look in to why this is actually needed because it is going to cause problems later on.